packages/block-library/src/image/index.php

```diff
@@ -191,6 +191,7 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 		'data-wp-context',
 		wp_json_encode(
 			array(
+				'animationId'      => uniqid(),
```

Comment:
We need to set a unique value in the context to make sure we only hide the content image that is currently being clicked. Is `uniqid()` a good way to do that?

Reply:
As this is more like unique id of the image, better name would be `imageId`

Reply:
`imageId` is also an existing attribute, so I would recommend using something more tied to the behavior of the overlay that is getting animated.